### ISSUE TITLE
'mmdetection/data/coco/annotations/instances_val2017.json' not found in annotation location

### ISSUE THREAD

**Issue by sunny1401** [fcjian/InstaGen#7]
Hi again - I am trying to fine-tune sdm, with novel categories. However, the annotation location only had train_instances, but in the .sh you add     ```--check_val_every_n_epoch 1 ```. Could you please tell what am I missing?

**Comment by fcjian** [comment]
@sunny1401 You should download the COCO dataset, and place it in the directory "mmdetection/data/coco"

**Comment by sunny1401** [comment]
Yes - I did add the image data from coco to the location - do you mean the annotations from validation are directly coming from coco? Asking cause you have added a [train annotations file](https://drive.google.com/file/d/1C41YDwP23Lh2pU33O_NNCaboMIdyt0Gv/view?usp=sharing)?

**Comment by fcjian** [comment]
@sunny1401 Yes, the annotation file of validation set is directly coming from  the COCO dataset. We specifically included the train annotation file because it is a new file that exclusively consists of annotations for the base categories.

**Comment by sunny1401** [comment]
Ah makes sense - thank you
